JSONModel/JSONModelTransformations/JSONKeyMapper.m

```diff
@@ -171,4 +171,45 @@ +(instancetype)mapperFromUpperCaseToLowerCase
 
 }
 
++ (instancetype)mapper:(JSONKeyMapper *)baseKeyMapper
+        withExceptions:(NSDictionary *)exceptions
+{
+    NSDictionary* userToModelMap = [exceptions copy];
+    NSDictionary* userToJSONMap  = [NSDictionary dictionaryWithObjects:exceptions.allKeys forKeys:exceptions.allValues];
```

Comment:
It means that it wouldn't be considered a breaking change to change the implementations in a way where the order became inconsistent. So yes, do need to switch to handling this safely.